specs/EmergencyAuthCodeEligibleOfficers.yml

```diff
@@ -3,12 +3,15 @@ info:
   title: Companies House Internal Oracle Query API
   version: '1.0'
 servers:
-  - url: http://localhost:18XXX
+  - url: http://localhost:18565
```

Comment:
Swagger doesn't like alpha characters in the port number - it messes up the test CURL statements it generates. Changed this to be the port used when service runs in Vagrant dev env.